app/src/main/res/values-night/themes.xml

```diff
@@ -36,6 +36,7 @@
         <item name="colorErrorContainer">@color/md_theme_dark_errorContainer</item>
         <item name="colorOnError">@color/md_theme_dark_onError</item>
         <item name="colorOnErrorContainer">@color/md_theme_dark_onErrorContainer</item>
+        <item name="colorBackground">@color/md_theme_dark_background</item>
```

Comment:
Why do you need to create custom attribute for background?

Reply:
Because I needed to get this color in the Config screen